test/cpp/rpc/test_tensorpipe_serialization.cpp

```diff
@@ -127,7 +127,7 @@ TEST(TensorpipeSerialize, NoDeleterTensors) {
   torch::distributed::rpc::TensorpipeWriteBuffers tpBuffers;
   std::tie(sendingTpMessage, tpBuffers) =
       torch::distributed::rpc::tensorpipeSerialize(
-          std::move(sendingRpcMessage));
+          std::move(sendingRpcMessage), {});
```

Comment:
There are a couple of callsites that pass in empty device indices, what's the semantics in that case (no device mapping is done)? If so, maybe we can put that as a default argument?

Reply:
Good point. In the current PR, is the map is not specified, users are only allowed to pass CPU tensors (same as the current behavior.)